examples/se2-car.cpp

```diff
@@ -5,66 +5,99 @@
 
 #include "aligator/solvers/proxddp/solver-proxddp.hpp"
 #include "aligator/core/explicit-dynamics.hpp"
+#include "aligator/modelling/composite-costs.hpp"
+#include "aligator/modelling/dynamics/integrator-euler.hpp"
 #include "aligator/modelling/quad-costs.hpp"
 #include "aligator/modelling/state-error.hpp"
-#include "aligator/modelling/composite-costs.hpp"
 #include "aligator/modelling/sum-of-costs.hpp"
 
 using T = double;
-using pinocchio::SpecialEuclideanOperationTpl;
-using proxsuite::nlp::PinocchioLieGroup;
-using SE2 = PinocchioLieGroup<SpecialEuclideanOperationTpl<2, T>>;
+using SE2 = proxsuite::nlp::SETpl<2, T>;
 
 using namespace aligator;
 using StateError = StateErrorResidualTpl<T>;
 using QuadResidualCost = QuadraticResidualCostTpl<T>;
+using context::StageModel;
+using context::TrajOptProblem;
+ALIGATOR_DYNAMIC_TYPEDEFS(T);
+
+struct CarDynamics : dynamics::ODEAbstractTpl<T> {
```

Comment:
You can write the function in latex to help people understand the model

Reply:
If you're motivated, you can also write the Ju, Jx equations (this will help to understand the matrix dimensions)

Reply:
Done! :)